pkg/identity/helpers.go

```diff
@@ -0,0 +1,85 @@
+package identity
+
+import (
+	"bytes"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/asn1"
+	"encoding/pem"
+	"errors"
+	"math/big"
+)
+
+func commonNameObjectIdentifier() asn1.ObjectIdentifier {
+	return asn1.ObjectIdentifier{
+		2, 5, 4, 3,
```

Comment:
what do these constants mean? same with below
would be nice to name these `const` if possible

Reply:
Yeah should definitely add some comments explaining these. Its a very weird part of the x509 spec that specifies what valid fields you can add to a issuer or subject identity in your cert. In Go, you can't set array constants unfortunately. Here are some examples of the entire OID tree: http://websites.umich.edu/~x509/ssleay/asn1-oids.html